ethpm_cli/config.py

```diff
@@ -81,6 +81,15 @@ def __init__(self, args: Namespace) -> None:
         else:
             self.project_dir = None
 
+        if "manifest_path" in args and args.manifest_path:
+            if not args.manifest_path.is_file():
+                raise ConfigurationError(
```

Comment:
I think you can do this at the parser level using this pattern

https://github.com/ethereum/trinity/blob/fc194a64215ccca1436f2ad874fcd244ce03db1f/trinity/cli_parser.py#L249-L279

Reply:
Is there an advantage to raising it at the parser level? Just curious since it seems like it'll require some extra configuration

Reply:
I believe the advantage is that this is a parser level error and there's a framework already in place via argparse for validating the user input and raising errors.  It's not necessarily better, but it can be good to have isolation, colocating all of a certain phase of validation in the same conceptual part of the application.